go.mod

```diff
@@ -67,8 +56,13 @@ replace (
 	// Terraform imports a pre-go-mod version of Vault. These replace directives
 	// resolves the ambiguous import between the package `vault/api` and
 	// `vault/api` nested go module.
-	github.com/hashicorp/vault => github.com/hashicorp/vault v1.5.5
+	github.com/hashicorp/vault => github.com/hashicorp/vault v1.7.0
 	github.com/hashicorp/vault/api => github.com/hashicorp/vault/api v1.0.5-0.20200805123347-1ef507638af6
-	github.com/hashicorp/vault/http => github.com/hashicorp/vault/http v1.5.5
-	github.com/hashicorp/vault/vault => github.com/hashicorp/vault/vault v1.5.5
+	github.com/hashicorp/vault/http => github.com/hashicorp/vault/http v1.7.0
+	github.com/hashicorp/vault/vault => github.com/hashicorp/vault/vault v1.7.0
+
+	// pin this version to avoid later versions that depend on a v1alpha1 branch that's no longer available
```

Comment:
suggestion to include in the comment that vault is project depending on the pinned k8s version (maybe that's obvious though) and that v0.18.2 is sourced from their go.mod

Reply:
Done!